.github/workflows/release-macos.yml

```diff
@@ -61,17 +61,17 @@ jobs:
              cd ..
       - name: Get runner environment variables
         id: runner
-        uses: msansoni/runner-environment-action@v1
+        uses: techno-express/action-environment-info@v1
```

Comment:
What does it mean here？

Reply:
the forked action from https://github.com/marketplace/actions/action-environment-info the other was not outputting OS version.